core/src/main/scala/org/apache/spark/ui/JettyUtils.scala

```diff
@@ -217,6 +220,9 @@ private[spark] object JettyUtils extends Logging {
       val pool = new QueuedThreadPool
       pool.setDaemon(true)
       server.setThreadPool(pool)
+      val errorHandler = new ErrorHandler();
```

Comment:
Sorry - should have been more explicit, but these should all be sans-colon. I can just merge this and fix it up.